README.md

```diff
@@ -1,20 +1,22 @@
 # Contents of this repository
 This repository contains software tools which allow to compare spectra using the Wasserstein distance and estimate relative abundances of molecules from the spectrum by minimizing the Wasserstein distance. 
 
-The tools are distributed as a Python3 package called `masserstein`. Basic functionality is also available as a set of commandline applications: `WSDistance` to compute the Wasserstein distance and `WSDeconv` to estimate proportions. 
+The tools are distributed as a Python3 package called `masserstein`. The package contains functions for generating spectra from chemical formulas, basic processing (centroiding, smoothing), and fitting combinations of reference spectra of analytes to a spectrum of their mixture.  
+Examples of use of the package functions are shown in the tutorials available on this repository.  
 
 If you encounter any difficulties during installation or usage of these programs, or if you have any suggestions regarding their functionality, please post a GitHub issue or send an email to m.ciach@mimuw.edu.pl. 
 
 # Installation
 
-To be able to use the software provided in this repository, you will need to have a working Python3 distribution installed on your computer. You will also need to have the following packages installed (availiable on PIP):
+To be able to use the software provided in this repository, you will need to have a working Python3 distribution installed on your computer.  
+The simplest way to install `masserstein` is to use the `pip` tool in the command-line: 
 
-* `IsoSpecPy`
-* `numpy`
-* `scipy`
-* `PuLP`
+```
+pip install masserstein
+```
 
-After installing these packages, download or clone this repository. In the commandline, this can be done by typing:
+However, this may not be the latest version of the package.  
```

Comment:
Don't write this way, it undermines your work. Masserstein on PyPI is in its stable version. Here on GitHub, a developmental version is maintained. So, remove this sentence and just: "To get a developmental version of the package, clone this repository" (remove this "download or").